### PR TITLE
Move CI to GitHub Actions

### PR DESCRIPTION
**Pull request by giordano**
Close #26

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,75 @@
+name: CI
+
+on:
+  push:
+    branches: "master"
+    tags: ["*"]
+  pull_request:
+  release:
```
Comment (mileslucas) [inline .github/workflows/ci.yml:8]:
```suggestion
on:
  push:
    branches: ["master"]
    tags: ["*"]
  pull_request:
    branches: ["master"]
```
IIRC releases automatically create tags, so there's no need to have _both_ in the CI.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
```
Comment (mileslucas) [inline .github/workflows/ci.yml:9]:
```suggestion
concurrency:
  # Skip intermediate builds: always.
  # Cancel intermediate builds: only if it is a pull request build.
  group: ${{ github.workflow }}-${{ github.ref }}
  cancel-in-progress: ${{ startsWith(github.ref, 'refs/pull/') }}
  
```
copied from the PkgTemplates CI

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+            ${{ runner.os }}-test-${{ env.cache-name }}-
+            ${{ runner.os }}-test-
+            ${{ runner.os }}-
+      - uses: julia-actions/julia-buildpkg@latest
+      - uses: julia-actions/julia-runtest@latest
+      - uses: julia-actions/julia-uploadcodecov@v0.1
+        continue-on-error: true
+      - uses: julia-actions/julia-uploadcoveralls@v0.1
+        continue-on-error: true
+
```
Comment (mileslucas) [inline .github/workflows/ci.yml:52]: I've also been adding a job to CI for aqua tests to catch things like missing compat, etc. if you want to add it! Here's a copy from another [JuliaAstro repo](https://github.com/JuliaAstro/LACosmic.jl/blob/76c67ff6a67147c5655538022f0220fac3993a19/.github/workflows/CI.yml#L52-L62) 
```suggestion
  aqua:
    name: Aqua tests
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - uses: julia-actions/setup-julia@v1
        with:
          version: '1'
      - uses: julia-actions/julia-buildpkg@v1
      - run: julia -e 'using Pkg; Pkg.add("Aqua")'
      - run: julia --project=@. -e 'using Aqua, UnitfulAstro; Aqua.test_all(UnitfulAstro, ambiguities=false)'
```

**Review by mileslucas — COMMENTED** [review]
oops too late

**Comment by giordano** [comment]
Mind doing a PR with the suggested changes? :grin:

**Comment by mileslucas** [comment]
I'll do that tomorrow!

**Pull request merged**